tools/mmp/Tuning.cs

```diff
@@ -131,6 +131,8 @@ static MonoMacLinkContext CreateLinkContext (LinkerOptions options, Pipeline pip
 			context.OutputDirectory = options.OutputDirectory;
 			context.StaticRegistrar = options.Target.StaticRegistrar;
 			context.Target = options.Target;
+			context.DisabledOptimizations |= CodeOptimizations.UnreachableBodies;  // TODO: revert this if linker issue is fixed
```

Comment:
A link to the linker issue in the comment would be nice.

Reply:
@akoeplinger whenever you are back, can we have the issue. Might have been fixed.